roles-in-tech/machine-learning.md

```diff
@@ -36,10 +39,35 @@
 
 ### Materials
 
+- [Example video (10 min)](https://example.com)
```

Comment:
I think this link should also be removed. Is it proper to remove? @saberprashant @alodahl

Reply:
Yeah.

Reply:
If this is appearing, you probably don’t have the right file...